lib/xero_gateway/oauth.rb

```diff
@@ -78,6 +80,22 @@ def renew_access_token(access_token = nil, access_secret = nil, session_handle =
       raise XeroGateway::OAuth::TokenInvalid.new(e.message)
     end
 
+    def get(path, headers = {})
```

Comment:
are these needed?

Reply:
these are actually consumed downstream of http.rb, by the formerly delegated fxns in this class (`XeroGateway::OAuth` is the `client` referenced in http.rb) - in order to inject additional headers, the delegation is replaced by these explicit implementations.

to trace how this currently works in the master branch:  an example regular api call (`get_contact`) would result in the following pseudo-trace:
```
xero_gateway/lib/xero_gateway/oauth.rb:26 (delegator to OAuth::AccessToken#get)
xero_gateway/lib/xero_gateway/http.rb:59:in `http_request'
xero_gateway/lib/xero_gateway/http.rb:8:in `http_get'
xero_gateway/lib/xero_gateway/gateway.rb:627:in `get_contact'
```